src/classes/Table/types.ts

```diff
@@ -106,7 +106,11 @@ export interface BatchGetOptions {
 }
 
 export interface BatchGetParamsMeta {
-  payload: any
+  payload: {
+    RequestItems: DocumentClient.BatchGetRequestMap;
+} & {
+    ReturnConsumedCapacity: string;
+} & Partial<DocumentClient.BatchGetItemInput>
```

Comment:
copy/paste from the inferred type of `payload` on [L1145](https://github.com/jeremydaly/dynamodb-toolbox/blob/7198da8e17b2a4113aa25892468a8d36eded5b27/src/classes/Table/Table.ts#L1144)